index.html

```diff
@@ -1,208 +1,380 @@
-        </p>
-        <ul>
-          <li><a href="">html</a></li>
-          <li><a href="">bootstrap</a></li>
-          <li><a href="">Ruby</a></li>
-        </ul>
-        <button>See Project</button>
-      </div>
-      <div class="card cd5">
-        <h3>Professional Art Printing Data</h3>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-          <li><a href="">html</a></li>
-          <li><a href="">bootstrap</a></li>
-          <li><a href="">Ruby</a></li>
-        </ul>
-        <button>See Project</button>
-      </div>
-      <div class="card cd6">
-        <h3>Professional Art Printing Data</h3>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul>
-          <li><a href="">html</a></li>
-          <li><a href="">bootstrap</a></li>
-          <li><a href="">Ruby</a></li>
-        </ul>
-        <button>See Project</button>
-      </div>
-    </section>
-    <section id="about">
-      <div class="about">
-        <h2>About me</h2>
-        <p>
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <button>Get My Resume</button>
-      </div>
-      <hr />
-      <div class="skillset">
-        <div class="specifics">
-          <h3><span>&#9830;</span>Language</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </div>
-        <div class="specifics">
-          <h3><span>&#9744;</span>Frameworks</h3>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>CapyBara</li>
-            <li>Salenium</li>
-          </ul>
-        </div>
-        <div class="specifics">
-          <h3><span>&#9679;</span>Skills</h3>
-          <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
-      </div>
-    </section>
-    <script src="script.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/dist/style.css" />

```

Comment:
- [ ] Kindly correct the path of the styles to show them, do not forget the dot at the beginning `./dist/style.css` 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/200112312-600ba2e8-dee7-4424-80f7-db505d0cce4a.png)